### PR TITLE
executor: remove useless test `TestSlidingWindowFunctionsSumDecimal`

### PR DESCRIPTION
**Pull request by glorv**
### What problem does this PR solve?
Test `TestSlidingWindowFunctionsSumDecimal` is a quite slow temp bench test case that cost nearly 3 minutes. according to @SunRunAway  and @mmyj , it can be removed.

### What is changed and how it works?
- remove test `TestSlidingWindowFunctionsSumDecimal`
- use common base struct to reduce code duplication for `testInfoschemaTableSuite` and avoid duplicated auto-generated tests

### Check List 

Tests 

- Unit test
- Integration test

### CODE REVIEW

**Comment by glorv** [comment]
@SunRunAway PTAL

File: executor/window_test.go [hunk 1]
```diff
@@ -311,8 +312,15 @@ func (s *testSuite7) TestSlidingWindowFunctionsSumDecimal(c *C) {
 	tk.MustExec("use test;")
 	tk.MustExec("drop table if exists t;")
 	tk.MustExec("CREATE TABLE t (id DECIMAL, sex CHAR(1));")
-	for i := 0; i < 100000; i++ {
-		tk.MustExec(fmt.Sprintf("insert into t values (%d,'M')", i%10))
+	var builder strings.Builder
```
Comment (SunRunAway) [inline executor/window_test.go]: After communicating with @mmyj, `TestSlidingWindowFunctionsSumDecimal` is useless now and can be removed.

**Comment by glorv** [comment]
@mmyj PTAL

**Pull request merged**